core/iwasm/common/gc/gc_common.c

```diff
@@ -14,15 +15,39 @@
 uint32
 wasm_get_defined_type_count(WASMModuleCommon *const module)
 {
-    /* TODO */
-    return 0;
+    WASMModule *wasm_module;
```

Comment:
```suggestion
```
WASMModule is only used when #if WASM_ENABLE_INTERP != 0, move this after line 21